h/accounts/test/models_test.py

```diff
@@ -6,20 +6,21 @@
 import pytest
 from sqlalchemy import exc
 
+from h import db
 from h.accounts import models
 from h.test import factories
 
 
-def test_activation_has_asciinumeric_code(db_session):
```

Comment:
So we can have _every_ test (even those that don't even use the db) do the savepoint and rollback with _no_ noticeable slowdown? Nice.

Not sure how I feel about the `db_session` being `autouse` vs explicit, I could go with it either way. I understand Robert's point but at the same time not having it `autouse` just introduces the possibility of forgetting to put the fixture on one test and introducing a test bug. Also this fixture is very generic code that should rarely need to be changed, or be the cause of a test failure.